src/ds/d400/d400-private.cpp

```diff
@@ -290,6 +290,84 @@ namespace librealsense
             return calc_intrinsic;
         }
 
+        //D405 needs special calculation because the ISP crops the full sensor image using non linear transformation.
+        rs2_intrinsics get_d405_color_stream_intrinsic(const std::vector<uint8_t>& raw_data, uint32_t width, uint32_t height)
+        {
+            // Convert normalized focal length and principal point to pixel units (K matrix format)
+            auto k_to_pixels = [&]( float3x3 & k, ds_rect_resolutions res )
```

Comment:
Please rename k_to_pixels to normalized_k_to_pixels

Reply:
Done